packages/amplify-graphql-transformer-core/src/graphql-api.ts

```diff
@@ -297,6 +297,7 @@ export class GraphQLApi extends GraphqlApiBase implements GraphQLAPIProvider {
     layers?: ILayerVersion[],
     role?: IRole,
     environment?: { [key: string]: string },
+    timeout?: Duration,
```

Comment:
Is this change related to the predictions directive?

Reply:
Yes. It's used [here](https://github.com/aws-amplify/amplify-cli/pull/7387/files#diff-e71c291954828fb3ee3f43caa98a7b430958d9e7816d2a28b0f49b64ebeb5824R407).